src/task.ts

```diff
@@ -74,23 +73,20 @@ export class Task<OutputType = unknown> implements QueueableTask {
   private readonly timeout?: number;
   private readonly startupTimeout?: number;
   private readonly maxRetries: number;
-  private readonly activityReadySetupFunctions: Worker<unknown>[];
-  private activity?: Activity;
-  private networkNode?: NetworkNode;
+  private readonly setupFunction?: LifecycleFunction;
```

Comment:
Seems to be unused, setup and teardown should be bound to the executor and not individual tasks